packages/ckeditor5-ui/src/toolbar/balloon/balloontoolbar.ts

```diff
@@ -115,8 +120,8 @@ export default class BalloonToolbar extends Plugin {
 
 		// Wait for the EditorUI#init. EditableElement is not available before.
 		editor.ui.once<EditorUIReadyEvent>( 'ready', () => {
-			this.focusTracker.add( editor.ui.getEditableElement()! );
 			this.focusTracker.add( this.toolbarView.element! );
+			this._trackEditablesFocusableElements();
```

Comment:
Can we set listeners before the editor is 'ready'? Maybe we could avoid initial-add and then listener-add?